tekton/resources/images/overlays/multi-arch/configuration.yaml

```diff
@@ -0,0 +1,9 @@
+nameReference:
+- kind: TriggerBinding
```

Comment:
NIT: There's no need to use kustomize on bindings, you can specify multiple bindings in a trigger, so for me the way to go would be to have the common base binding and another platform binding which is only used on the multi-arch trigger.